client/rest/src/main/java/org/elasticsearch/client/HeapBufferedAsyncResponseConsumer.java

```diff
@@ -67,6 +67,13 @@ public HeapBufferedAsyncResponseConsumer(int bufferLimit) {
         this.bufferLimit = bufferLimit;
     }
 
+    /**
+     * Get the limit of the buffer.
+     */
+    public int getBufferLimit() {
```

Comment:
I think changing it to a `ByteSizeValue` would be a fairly big project because `ByteSizeValue` is in elasticsearch and this is in the client and they don't share code at this point. They *should* be able to at some point, but they don't at this point. So `getBufferLimitBytes` it is, I think.